cms/djangoapps/contentstore/views/course.py

```diff
@@ -896,6 +915,42 @@ def _config_course_advanced_components(request, course_module):
     return filter_tabs
 
 
+# pylint: disable=invalid-name
+def _config_course_settings(request, course_module, filter_tabs=True):
+    """
+    Check to see if the user enabled some advanced settings (boolean).
```

Comment:
> I don't understand why there is a second method (which looks very much like _config_course_advanced_components). Please explain why there are 2 methods, and make sure the comments in this method are all correct (there are references to advanced settings, but I don't think this method is supposed to be about advanced settings).

[_config_course_advanced_components](https://github.com/edx/edx-platform/blob/feature/edxnotes/cms/djangoapps/contentstore/views/course.py#L872) is more specific and works only with advanced_modules field. It adds/removes appropriate tab when specific module is added.
[_config_course_settings](https://github.com/edx/edx-platform/blob/feature/edxnotes/cms/djangoapps/contentstore/views/course.py#L919) has the similar logic, but works with course fields.

We'll update names and comments.